docs/client-libraries.md

```diff
@@ -10,7 +10,7 @@ nav = [
 
 ## Client Libraries
 
-This section contains example code for client libraries in several languages. Please remember that when modeling your data in Pilosa, it is best to keep row and column ids sequential. It is not wise to use the output of a hash, or randomly distributed ids with Pilosa.
+This section contains example code for client libraries in several languages. Please remember that when modeling your data in Pilosa, it is best to keep row and column ids sequential. It is best to avoid using the the output of a hash, or randomly distributed ids with Pilosa.
```

Comment:
the comma after "hash" is not necessary, unless you also put a comma after "ids".